src/pug/includes/body.pug

```diff
@@ -18,7 +18,7 @@ include ./fork.pug
                         label.sr-only(for="filter") Filter opportunity difficulty
                         select#filter
                             for filter in ['All difficulties', 'Easy', 'Medium', 'Hard']
-                                option(value=filter.replace(/ /g, '').toLowerCase()) #{filter}
+                                option(value=filter.split(' ')[0].toLowerCase()) #{filter}
```

Comment:
Sorry if I wasn't clear, I was suggesting the UI be `select[all, easy, medium, hard] difficulties`

so when you chose medium it would effectively say `medium difficulties` rather than just medium which loses context

Reply:
I think we should `difficulties` can be moved outside the select